MozillaFoundation/foundation.mozilla.org
# Highlights in preview panel not updating

# Description

This PR retains the sort_order of the ordered highlights method but updates it to query from the in-memory model instance via `self` instead of using the ORM, enabling preview panel to update in real time as highlights are reordered on the edit screen.

Link to sample test page: https://foundation-s-p1-150-105-uva1ex.herokuapp.com/cms/pages/11/edit/
Related PRs/issues: #10515

username: admin
password: h04MgDCy$rCloFYl

# To Test
1) Log in to CMS
2) Go to sample test page
3) Move highlight ordering
4) Preview panel should respect highlight ordering

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-500)
